src/components/shared/Project/VolumeStatLine/VolumeStatLine.tsx

```diff
@@ -0,0 +1,70 @@
+import { Trans } from '@lingui/macro'
+import ETHAmount from 'components/shared/currency/ETHAmount'
+import CurrencySymbol from 'components/shared/CurrencySymbol'
+
+import { ThemeContext } from 'contexts/themeContext'
+import { Property } from 'csstype'
+import { BigNumber } from 'ethers/lib/ethers'
```

Comment:
**suggestion**: so treeshaking kicks in, we should use

```suggestion
import { BigNumber } from '@ethersproject/bignumber'
```

Reply:
nice! added